### DISCUSSION TITLE
Camera pan problem on iOS/iPadOS PWA

### DISCUSSION THREAD

**Discussion by DanieleMagri** [google/model-viewer#3593] [Q&A]
Problem:
I touch the screen, hold and drag to rotate the model.
The model starts to rotate following the finger movement correctly, but freezes for no reason after a while, sometimes it doesn't even detect touch, even the UI seems to freeze when this happens.
Freeze only happen when interacting with the model, not the UI.
This problem only occurs if the website is installed from Safari on the device home, as a Web App.
If used directly on Safari, without installing it as a PWA, the problem does not occur.

Tested on:
iPad Pro 12.9 (M1)
iPhone SE 2

**Comment by elalish** [comment]
That's very interesting, but sounds like you should file the bug on WebKit. Please keep us updated if you find out why, or if there's anything we should do to avoid it.
